### PR TITLE
Throws a bunch of parenthesis around to ensure dear Aunt Sally is always properly excused.

### PR DESCRIPTION
**Pull request by ShizCalev**
Similar vein to #37116

This is supposed to be standard, yet here we are.

SHOULDN'T change anything, but there's likely something out there that's bound to behave different because of it. 

These were done manually, regex to find things that MIGHT need to be corrected;
`^#define.+\+((?!\)).)*$`
`^#define.+-((?!\)).)*$`
`^#define.+\*((?!\)).)*$`
`^#define.+\/((?!\)).)*$` (yeah that's a lot of stuff.)
`^#define.+%((?!\)).)*$`
`^#define.+SECONDS((?!\)).)*$`
`^#define.+MINUTES((?!\)).)*$`

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
It was recently added to our standards I think

File: code/__DEFINES/blob_defines.dm [hunk 1]
```diff
@@ -3,11 +3,11 @@
 #define OVERMIND_MAX_POINTS_DEFAULT 100 // Max point storage
 #define OVERMIND_STARTING_POINTS 60 // Points granted upon start
 #define OVERMIND_STARTING_REROLLS 1 // Free strain rerolls at the start
-#define OVERMIND_STARTING_MIN_PLACE_TIME 1 MINUTES // Minimum time before the core can be placed
-#define OVERMIND_STARTING_AUTO_PLACE_TIME 6 MINUTES // After this time, randomly place the core somewhere viable
+#define OVERMIND_STARTING_MIN_PLACE_TIME (1 MINUTES) // Minimum time before the core can be placed
+#define OVERMIND_STARTING_AUTO_PLACE_TIME (6 MINUTES) // After this time, randomly place the core somewhere viable
```
Comment (LemonInTheDark) [inline code/__DEFINES/blob_defines.dm:7]: this seems silly

File: code/__DEFINES/qdel.dm [hunk 2]
```diff
@@ -31,9 +31,9 @@
 #define GC_QUEUE_COUNT 3 //! Number of queues, used for allocating the nested lists. Don't forget to increase this if you add a new queue stage
 
 // Defines for the time an item has to get its reference cleaned before it fails the queue and moves to the next.
-#define GC_FILTER_QUEUE 1 SECONDS
-#define GC_CHECK_QUEUE 5 MINUTES
-#define GC_DEL_QUEUE 10 SECONDS
+#define GC_FILTER_QUEUE (1 SECONDS)
+#define GC_CHECK_QUEUE (5 MINUTES)
+#define GC_DEL_QUEUE (10 SECONDS)
```
Comment (LemonInTheDark) [inline code/__DEFINES/qdel.dm:36]: this too

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]
seems odd to apply this to seconds defines

**Comment by ShizCalev** [comment]
Do you want me to remove all of these that presently exist in the code then?
image
image

Seems weirder to cherry pick that one define when ultimately it's still performing math.

**Comment by LemonInTheDark** [comment]
There aren't many operators with higher priority then *, and it seemed strange to apply the pattern to seconds instances knowing that.
I guess there's **, so it could save some suffering

**Comment by JohnFulpWillard** [comment]
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#put-macro-segments-inside-parentheses-where-possible

**Comment by Mothblocks** [comment]
i think its worth being consistent for seconds

**Pull request merged**